src/test/java/com/rcv/TabulatorTests.java

```diff
@@ -138,6 +146,18 @@ private static void runTabulationTest(String stem) {
     }
   }
 
+  private static void compareJson(
+      ContestConfig config, String stem, String timestampString, Integer sequentialNumber) {
+    String actualOutputPath =
+        ResultsWriter.getSummaryOutputPath(
+            config.getOutputDirectory(), timestampString, sequentialNumber)
+            + ".json";
+    String expectedPath =
+        getTestFilePath(
+            stem, (sequentialNumber != null ? "_" + sequentialNumber : "") + "_expected.json");
```

Comment:
is there a way to avoid duplicating this file name generating code?

Reply:
Do you mean specifically the part that appends an underscore and the sequential number? OK, I can add a one-line static method in `ResultsWriter`.